docs/notebooks/index.rst

```diff
@@ -0,0 +1,13 @@
+Examples
+========
+
+.. toctree::
+    :maxdepth: 1
+
+    1_catalog
+    2_getting_started
+    3_diagnostics
+    4_ensemble_reduction
+    5_warminglevels
+    6_config
+    7_global_tas_average_obs
```

Comment:
I don't think we want that one in the doc? It wasn't there before.

Reply:
If we don't want it, I'll remove it.

Reply:
I would remove, it doesn't use xscen. It is used to build xscen. @aulemahal did you want it in the doc?

Reply:
I put it there so it's not lost and we can reproduce the work, but it is not meant to be in the main documentation!